package.json

```diff
@@ -18,13 +18,7 @@
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
     "express": "^4.17.2",
-    "install": "^0.13.0",
```

Comment:
I didn't see that any of these were used, so I just removed them, but you could also add them to .serverlessignore, you just have to be mindful of transitive dependencies.